ui/ui.go

```diff
@@ -96,27 +96,38 @@ func (ui *WriterUI) PrintTable(table Table) {
 	}
 }
 
-func (ui *WriterUI) AskForText(label string) (string, error) {
-	var text string
+func (ui *WriterUI) AskForText(opts TextOpts) (string, error) {
+	text := opts.DefaultValue
 
-	err := interact.NewInteraction(label).Resolve(&text)
+	err := interact.NewInteraction(opts.Label).Resolve(&text)
 	if err != nil {
 		return "", fmt.Errorf("Asking for text: %s", err)
 	}
 
 	return text, nil
 }
 
-func (ui *WriterUI) AskForChoice(label string, options []string) (int, error) {
-	var choices []interact.Choice
+func (ui *WriterUI) AskForChoice(opts ChoiceOpts) (int, error) {
+	var (
+		choices                    []interact.Choice
+		defaultMatchingWithChoices bool
+		chosen                     int
+	)
 
-	for i, opt := range options {
+	for i, opt := range opts.Choices {
+		if opt == opts.DefaultValue {
+			chosen = i
+			defaultMatchingWithChoices = true
+		}
 		choices = append(choices, interact.Choice{Display: opt, Value: i})
 	}
 
-	var chosen int
+	if !defaultMatchingWithChoices {
```

Comment:
`default` can be empty string as well, which is a valid use case. I think we should change the condition to `!defaultMatchingWithChoices && len(default) !=0`

Reply:
Adopted.